### ISSUE TITLE
Is there a 32-bit ISO file?

### ISSUE THREAD

**Issue by qwIvan** [ric96/atvx86#7]

**Comment by kyvaith** [comment]
Try this: https://mega.nz/#!IsE0XJDA!yFTqfj0Vf-IAdiXRolSbGjnbWfz9eGGCrT8QNAWaTZw It's a mess, so don't be to excited. I'm working on doing this properly.

**Comment by qwIvan** [comment]
@kyvaith 
Where is it from? Is it built by yourself?

**Comment by kyvaith** [comment]
Yes, its my build. Have you tried it on native hardware? Do you have YouTube working?

**Comment by isosa28** [comment]
Hello Kyvaith,
I've try your build on a virtual machine and youtube works and all apps.
I also try it on my dell 7548 and all works except WIFI, do you know how to get wifi working?
